multisig-wallet/Forc.lock

```diff
@@ -0,0 +1,12 @@
+[[package]]
```

Comment:
Hmm. Actually, since we're using tagged releases of the compiler in CI, how about deleting Forc lockfiles and adding `Forc.lock` to the `.gitignore`? Since these projects are meant to showcase maintained examples rather that being deployed to production as-is, keeping up with dependency updates is a desirable property.